subscribe-api.md

```diff
@@ -36,7 +36,55 @@ Parameter | Required  | Value
 `connectionId` | Mandatory | The same `connectionId` that is generated with the Streaming API can be used to subscribe to this API.  
 `accessToken` | Mandatory | The Symbl's Authentication token you get from the [Authentication process](/docs/developer-tools/authentication/).
 
+### Request Body
+
+``` js
+const WebSocket = require('ws');​
+const accessToken = "<Your Token>"
```

Comment:
We should follow a coding style guide across our codes. Over here the semicolon `;` isn't present but is present at other places. https://airbnb.io/javascript/ this is one of the most used style guides in the industries at the moment.